lms/envs/common.py

```diff
@@ -2530,6 +2530,20 @@
 TIME_ZONE_DISPLAYED_FOR_DEADLINES = 'UTC'
 
 
+########################## VIDEO IMAGE STORAGE ############################
+
+VIDEO_IMAGE_SETTINGS = dict(
+    # Backend storage
+    # STORAGE_CLASS='storages.backends.s3boto.S3BotoStorage',
+    # STORAGE_KWARGS=dict(bucket='video-image-test-bucket'),
```

Comment:
yes. just a convention to tell what the actual values look like. I will prefer to keep the dict constructor format.